tensorboard/webapp/runs/views/runs_table/runs_group_menu_button_component.ts

```diff
@@ -31,9 +33,21 @@ import {GroupBy, GroupByKey} from '../../types';
 export class RunsGroupMenuButtonComponent {
   readonly GroupByKey = GroupByKey;
 
+  @Input() experimentIds!: string[];
+  @Input() regexString!: string;
   @Input() selectedGroupBy!: GroupBy;
   @Input() showGroupByRegex!: boolean;
 
   @Output()
   onGroupByChange = new EventEmitter<GroupBy>();
+
+  constructor(public dialog: MatDialog) {}
+
+  onRegexStringEdit() {
+    // data pass in the experiment id
+    const dialogRef = this.dialog.open(RegexEditDialogContainer, {
+      width: '1050px',
```

Comment:
Is there a way to alter the width and height? Seems like it is possible to set `maxHeight` and `maxWidth`: https://material.angular.io/components/dialog/api#MatDialogConfig

Reply:
ah, sorry I forget about this. updated.